diff --git a/node_modules/@fullcalendar/common/main.js b/node_modules/@fullcalendar/common/main.js
index b4a04e7..1a35c00 100644
--- a/node_modules/@fullcalendar/common/main.js
+++ b/node_modules/@fullcalendar/common/main.js
@@ -3,7 +3,7 @@ FullCalendar v5.5.1
 Docs & License: https://fullcalendar.io/
 (c) 2020 Adam Shaw
 */
-import './main.css';
+//import './main.css';
 
 import { __assign, __spreadArrays, __extends } from 'tslib';
 import { createContext, Component, createRef, createElement, Fragment } from './vdom.js';
